### PR TITLE
#23181339489 Login via Facebook and Google implementation

### PR DESCRIPTION
**Pull request by Tsa2341**
#### What does this PR do?

  Implement the login with google and facebook

#### Description of Task to be completed?

- user can log in to the barefoot application with google and Facebook

#### How should this be manually tested?
here is the deployed link: [https://deploy-preview-10--barefoot-nomad-fe.netlify.app/login](https://deploy-preview-10--barefoot-nomad-fe.netlify.app/login)

#### What are the relevant pivotal tracker stories?

- [#181339489](https://www.pivotaltracker.com/story/show/181339489)

### CODE REVIEW

**Review by Gabin-ishimwe — APPROVED** [review]

**Review by Code-250 — APPROVED** [review]

**Review by ncutixavier — APPROVED** [review]

File: src/view/login.js [hunk 1]
```diff
@@ -275,18 +276,30 @@ const Login = () => {
...
+                `${process.env.DEPLOY_PRIME_URL}/social/login`,
+              )}`}
+            >
+              <Buttons
+                variant="contained"
+                startIcon={<FacebookRoundedIcon />}
+                sx={mediaStyles}
+                value="Login with Facebook"
+              />
+            </A>
```
Comment (ncutixavier) [inline src/view/login.js:302]: You must use redux. it's mandatory.

**Review by ncutixavier — CHANGES_REQUESTED** [review]

File: src/view/login.js (hunk 1 above)
Comment (Tsa2341) [reply src/view/login.js:302]: implemented it in google_login.js component

**Comment by ncutixavier** [comment]
Fix code climate issues

**Comment by ncutixavier** [comment]
Don't put your redux files in login, they have to be separated

**Review by ncutixavier — COMMENTED** [review]
check your redirect URL after sign-in

**Review by ncutixavier — APPROVED** [review]

**Pull request merged**